pkg/operator/ceph/config/livenessprobe.go

```diff
@@ -51,3 +51,28 @@ func ConfigureLivenessProbe(daemon rookv1.KeyType, container v1.Container, healt
 
 	return container
 }
+
+func GetLivenessProbeWithDefaults(desiredProbe, defaultProbe *v1.Probe) *v1.Probe {
```

Comment:
Actually, it would have been nice to have unit tests for this so we can verify the behavior remains consistent.

Reply:
Added unit test.